src/utils/proxy.mjs

```diff
@@ -157,12 +157,21 @@ const serveRedirect = async function ({ match, options, proxy, req, res, siteInf
   }
 
   if (match.signingSecret) {
-    req.headers['x-nf-sign'] = signRedirect({
-      deployContext: 'dev',
-      secret: match.signingSecret,
-      siteID: siteInfo.id,
-      siteURL: siteInfo.url,
-    })
+    const signingSecretVar = env[match.signingSecret]
+
+    if (signingSecretVar) {
+      req.headers['x-nf-sign'] = signRedirect({
+        deployContext: 'dev',
+        secret: signingSecretVar.value,
```

Comment:
is there a chance `signingSecretVar` will ever be falsy or not contain a `value`?